### ISSUE TITLE
Suppress output from build (maven)

### ISSUE THREAD

**Issue by yusijs** [khalilou88/jnxplus#99]
Hi! 

I might have missed something here, apologies if so. 

On serve we have the "options" flag which allows us to pass arguments to maven, e.g:
```json
      "options": {
        "args": "-Dspring-boot.run.profiles=local"
      }
```

However for build there is no options / args. This means I'm seeing 20-30 000 lines of "Downloading" logs in github actions, and I would be ecstatic if I somehow could get rid of those :)

**Comment by khalilou88** [comment]
Hi @yusijs  you are right we don't have this option for build executor. The idea is to start with small features and add more when it's needed. What you are asking is a valid use case but maybe instead of adding an args we can use the nx "Verbose" mode.
I will dig more

**Comment by khalilou88** [comment]
Agree and more simple, could you please make a Pull_request and I will merge it?

**Comment by yusijs** [comment]
https://github.com/khalilou88/jnxplus/pull/100

**Comment by khalilou88** [comment]
Thanks just I want to change the name to mvnArgs, because I will added it other executors later, "args" for serve was for spring boot I think

**Comment by yusijs** [comment]
What about gradle? gradleArgs maybe?

**Comment by khalilou88** [comment]
yes please

**Comment by khalilou88** [comment]
@yusijs I need to do more test maybe lets keep it simple et keep only one args

**Comment by yusijs** [comment]
Sorry, you want to keep it as args instead of maven/gradleArgs then?

**Comment by khalilou88** [comment]
@yusijs yes, I will add some tests and if args is good enough I will publish a new version

**Comment by khalilou88** [comment]
@yusijs I published the version @jnxplus/nx-boot-maven@3.0.0 with mvnArgs option for build executor. Hope it will fix your issue

**Comment by yusijs** [comment]
> @yusijs I published the version `@jnxplus/nx-boot-maven@3.0.0` with mvnArgs option for build executor. Hope it will fix your issue

Awesome, thanks :)

**Comment by khalilou88** [comment]
@yusijs I recommend now using the  env var NX_MAVEN_CLI_OPTS for global options (starting from version 6.2.0 for nx-boot-maven).
You can use it like that     
 env:
          NX_MAVEN_CLI_OPTS: '--no-transfer-progress'
